tests/TestHelpers/HttpLogger.php

```diff
@@ -0,0 +1,124 @@
+<?php declare(strict_types=1);
+/**
+ * ownCloud
+ *
+ * @author Sajan Gurung <sajan@jankaritech.com>
+ * @copyright Copyright (c) 2023 Sajan Gurung sajan@jankaritech.com
```

Comment:
```suggestion
 * @copyright Copyright (c) 2023, ownCloud GmbH
```
I think we can be an author, but we don't have copyright rights, that must be the owncloud.